.github/ISSUE_TEMPLATE/4_decommission-hub.md

```diff
@@ -43,7 +43,7 @@ Usually, it is because it was a hub that we created for a workshop/conference an
 _This phase is only necessary for single hub clusters._
 
 - [ ] Run `terraform plan -destroy` and `terraform apply` from the [appropriate workspace](https://infrastructure.2i2c.org/en/latest/topic/terraform.html#workspaces), to destroy the cluster
-- [ ] Remove the following files from the repository:
-  - The associated `config/clusters/<cluster_name>` directory
-  - The associated CI deployer key in `secrets/`
-  - Remove the name of the cluster from [CI](https://github.com/2i2c-org/infrastructure/tree/HEAD/.github/workflows/deploy-hubs.yaml)
+- [ ] Remove the associated `config/clusters/<cluster_name>` directory and all its contents
+- Remove the cluster from CI:
+  - [ ] [`deploy-hubs.yaml`](https://github.com/2i2c-org/infrastructure/blob/HEAD/.github/workflows/deploy-hubs.yaml)
+  - [ ] [`validate-clusters.yaml`](https://github.com/2i2c-org/infrastructure/blob/HEAD/.github/workflows/validate-clusters.yaml)
```

Comment:
@GeorgianaElena I'm wondering if there's any steps we should add here relating to the central grafana? Or is it too early to tell?

Reply:
Hmm, good point! Currently, we have the script at https://github.com/2i2c-org/infrastructure/blob/master/deployer/update_central_grafana_datasources.py that adds each cluster as a datasource to the central grafana if it isn't already. We should add an option to this script to delete a datasource if it isn't in the cluster list anymore.

I will open an issue about this, but maybe until then, we can add a bullet point that says, `Remove the cluster from the list of datasources at https://grafana.pilot.2i2c.cloud/datasources`?

Reply:
Done in [9299b6c](https://github.com/2i2c-org/infrastructure/pull/1457/commits/9299b6c451bea3a0079cc9e2ad30f5b3a1a83eda)